content/en/docs/how-tos/rbac.md

```diff
@@ -44,6 +44,57 @@ No API is provided to create Rover groups. We have to do some manual work on the
 The existing groups defined in the release repo will be removed from the release repo and the clusters. 
 If such a group is used as a subject in a RBAC manifest, then its owner has to ensure the group also exists on Rover. In case that the group name cannot be managed by the owner, e.g., the desired group name is taken by other teams, then the RBAC manifest has to use another Rover group.
 
+For example, `group/cvp-pool-admins` cannot exist any more in the release repo because it contains usernames.
+
+```yaml
+apiVersion: v1
+kind: List
+items:
+- apiVersion: user.openshift.io/v1
+  kind: Group
+  metadata:
+    name: cvp-pool-admins
+  users:
+  - bob
+  - jim
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: cvp-pool-admins
+    namespace: cvp-cluster-pool
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: hive-cluster-pool-admin
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: cvp-pool-admins
+```
+
+When it is removed from the release repo and the cluster, the members of the group lose the corresponding permissions on the clusters.
+To retrieve the permissions, we need
+
+- create a group `cvp-pool-admins` at [Rover](https://rover.redhat.com/groups/) with the same members, and
```

Comment:
I am wondering if we could (should?) recommend our users to not think of Rover Groups as entities to be synced to clusters, but primarily as groups of people who should somehow share similar levels of access. So in this specific example, I would not recommend people to create a Rover Group called `cvp-pool-admins` (because that ties the name of the rover group to one specific application), but I would recommend them to create a Rover Group for their team (or even better - maybe they already have one) called `cvp-team-members` or the like, and then use *that* group in possibly multiple places in OpenShift CI.

Note that we did not follow this principle ourselves, it seems. I looks like we have `test-platform-ci-admins` and `test-platform-ci-sudoers` that mixup TP members and external engineers, so when a new TP engineer joins the team, we need to add them to multiple places over time. I think we should have just a single group that represents our team, called `test-platform-team-members`, and use that one single group everywhere. Then, we can have smaller specialized groups for specialized levels of access for external engineers.

Reply:
Let me start to explain the 2nd paragraph above:

We have 3 groups for now. I was planning that we manage only those 3.
`test-platform-ci-admins`: Basically they are DP members. They are admins on each cluster.

`test-platform-ci-sudoers`: includes the people helping us debugging clusters. They are admins on chosen clusters, currently only on build01 and build02. Why are dp members in the group too? That is a Rover thing: Each DP member should have access to manage the Rover group (owners of the group). I do not want any other groups to be sudoers on our cluster. Ideally, it should be defined as list of other-sudoers (members of the group) + test-platform-ci-admins (owners of the group) instead of re-listing every member of test-platform-ci-admins. But I cannot express group inclusion via Rover.

`test-platform-ci-monitoring-viewers`: people to view our monitoring stack. I created it on Rover only because that I want to keep the current users access. We do not have manage this group at all because I do not see any risk. This group is optional to me in a long run. Only needed for transitional phase. I could remove this now and expect the users to add their groups back as for other bindings if you prefer that way.

About the first paragraph:
We leave choice of group to the users:
1. Either create a group with the same name as in binding. No need to change the binding subject this way, or,
2. Use a preferred group (existing or create if not on Rover). Rename it by the config file or modify the subject in bindings.

It seems you prefer our users to do 2. Let me make that clear in the doc.
I do not have any preference.

Reply:
Updated.